include/stl2/detail/iterator/ostreambuf_iterator.hpp

```diff
@@ -23,69 +23,68 @@
 #include <stl2/detail/iterator/default_sentinel.hpp>
 
 STL2_OPEN_NAMESPACE {
-	namespace __ostreambuf_iterator {
-		template <MoveConstructible charT, class traits>
-		class cursor {
-			using ostream_type = std::basic_ostream<charT, traits>;
-			using streambuf_type = std::basic_streambuf<charT, traits>;
-			detail::raw_ptr<streambuf_type> sbuf_ = nullptr;
-
+	// Not to spec:
+	// * MoveConstructible requirement is implicit
+	//   See https://github.com/ericniebler/stl2/issues/246)
+	// * Extension: satisfies EqualityComparable and Sentinel<default_sentinel>
+	template <MoveConstructible charT, class traits = std::char_traits<charT>>
+	class ostreambuf_iterator {
 		public:
-			class mixin : protected detail::ebo_box<cursor> {
-				using box_t = detail::ebo_box<cursor>;
-			public:
-				using difference_type = std::ptrdiff_t;
-				using char_type = charT;
-				using traits_type = traits;
-				using ostream_type = cursor::ostream_type;
-				using streambuf_type = cursor::streambuf_type;
-
-				constexpr mixin() noexcept = default;
-				using box_t::box_t;
+			typedef ptrdiff_t difference_type;
+			typedef charT char_type;
+			typedef traits traits_type;
+			typedef std::basic_streambuf<charT, traits> streambuf_type;
+			typedef std::basic_ostream<charT, traits> ostream_type;
```

Comment:
Puke, typedefs again. If you really want these to exactly match the text in the TS, I would be happy to editorially change them to using aliases in the TS - which I think is what happened to the C++ WP wording.